mathics/builtin/drawing/uniform_polyhedra.py

```diff
@@ -7,32 +7,40 @@
 and regular star polyhedra.
 """
 
+from mathics.core.builtin import Builtin
+from mathics.core.evaluation import Evaluation
+
 # This tells documentation how to sort this module
 # Here we are also hiding "drawing" since this can erroneously appear at the top level.
 sort_order = "mathics.builtin.uniform-polyhedra"
 
-from mathics.core.builtin import Builtin
-from mathics.core.evaluation import Evaluation
-
 uniform_polyhedra_names = "tetrahedron, octahedron, dodecahedron, icosahedron"
 uniform_polyhedra_set = frozenset(uniform_polyhedra_names.split(", "))
 
 
 class Dodecahedron(Builtin):
     """
-    <url>:WMA link:
-    https://reference.wolfram.com/language/ref/Dodecahedron.html</url>
+
+    <url>:Dodecahedron:
+    https://en.wikipedia.org/wiki/Dodecahedron</url> (<url>:WMA:
+    https://reference.wolfram.com/language/ref/Dodecahedron.html</url>)
 
     <dl>
       <dt>'Dodecahedron[]'
       <dd>a regular dodecahedron centered at the origin with unit edge length.
+
+      <dt>'Dodecahedron[$l$]'
```

Comment:
<dt>'Dodecahedron'[$1$]

Reply:
It is "l" for length. See https://reference.wolfram.com/language/ref/Dodecahedron.html or the "dd" description.